### DISCUSSION TITLE
Classify exporters using IP CIDR

### DISCUSSION THREAD

**Discussion by eahlys** [akvorado/akvorado#168] [Q&A] [answered]
Hi,
I would like to classify the site of every exporter (`ClassifySite()`) using their IPv4 addresses. Is there a better way to do this than regexes ? I have a few /18 I want to classify (they are not easy /16 that I can classify using startsWith)

Thanks :)

**Comment by vincentbernat** [comment, marked as answer]
There is nothing better yet. However, it does not seem hard to add a `Subnet` variant (like the `Regex` variant). If you know some Go, you can try it yourself (it's in `inlet/core/classifier.go`).
